Java/formatter/intellij-java-google-style-pre-14.1.xml

```diff
@@ -12,8 +12,8 @@
       <option name="USE_RELATIVE_INDENTS" value="false" />
     </value>
   </option>
-  <option name="CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" value="99" />
-  <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="99" />
+  <option name="CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" value="4" />
```

Comment:
Because it's higher than 3, but less than 5?

I guessed that 4 means "collapse imports >= 4" rather than >4, with the intent to check whether that was right next time I was in IDEA. I'll do that after lunch.